crates/turbopack-ecmascript/tests/tree-shaker/analyzer/write-order/output.md

```diff
@@ -0,0 +1,376 @@
+# Items
+
+Count: 11
+
+## Item 1: Stmt 0, `VarDeclarator(0)`
+
+```js
+export const order = [];
+
+```
+
+- Declares: `order`
+- Write: `order`
+
+## Item 2: Stmt 1, `Normal`
+
+```js
+export function func() {
+    order.push("d");
+}
+
+```
+
+- Hoisted
+- Declares: `func`
+- Reads (eventual): `order`
+- Write: `func`
+
+## Item 3: Stmt 2, `Normal`
+
+```js
+order.push("a");
+
+```
+
+- Side effects
+- Reads: `order`
+
+## Item 4: Stmt 3, `VarDeclarator(0)`
+
+```js
+const x1 = externalFunction();
+
+```
+
+- Declares: `x1`
+- Reads: `externalFunction`
+- Write: `x1`
+
+## Item 5: Stmt 4, `VarDeclarator(0)`
+
+```js
+const x2 = externalFunction();
+
+```
+
+- Declares: `x2`
+- Reads: `externalFunction`
+- Write: `x2`
+
+## Item 6: Stmt 5, `VarDeclarator(0)`
+
+```js
+export const shared = {
+    effect: order.push("b")
+};
+
+```
+
+- Declares: `shared`
+- Reads: `order`
+- Write: `shared`
+
+## Item 7: Stmt 6, `Normal`
+
+```js
+order.push("c");
+
+```
+
+- Side effects
+- Reads: `order`
```

Comment:
```suggestion
- Side effects
- Writes: `order`
```

Technically it doesn't need to be flagged as `Side Effects`, but I doubt we can detect that.